db/change_cache_test.go

```diff
@@ -1406,7 +1406,7 @@ func TestLateArrivingSequenceTriggersOnChange(t *testing.T) {
 		CollectionID: collectionID,
 		DataType:     dataType,
 	}
-	db.changeCache.DocChanged(feedEventDoc2)
+	db.changeCache.DocChanged(feedEventDoc2, 0)
```

Comment:
Should use the const here instead of `0`, since we know we're writing an actual customer document.


```suggestion
	db.changeCache.DocChanged(feedEventDoc2, DocTypeDocument)
```